leshan-server-cf/src/main/java/org/eclipse/leshan/server/californium/impl/CaliforniumLwM2mRequestSender.java

```diff
@@ -61,16 +64,16 @@ public CaliforniumLwM2mRequestSender(final Set<Endpoint> endpoints, final Observ
     }
 
     @Override
-    public <T extends LwM2mResponse> T send(final Client destination, final DownlinkRequest<T> request, Long timeout)
-            throws InterruptedException {
+    public <T extends LwM2mResponse> T send(final Client registrationInfo, final DownlinkRequest<T> request,
```

Comment:
I think we should stick with the original name of the parameter but (soon) change the name of `Client` to `RegistrationInfo` instead. The parameter object in this case still plays the role of the *destination*, doesn't it?